apps/drivers.inc

```diff
@@ -13,6 +14,11 @@ TTY_CONST = 0
 TTY_CONIN = 1
 TTY_CONOUT = 2
 
+AUX_IN = 0

```

Comment:
I actually want to rework this into a generic SERIAL driver, supporting things like changing baud rate, but I'll do that myself and make sure it's binary compatible with this.

Reply:
That's also a good idea, you can add a "transparent"- mode to the consoleIO. My initial thoughts are, that systems with an extra serial-interface could use "AUX" , but that doesn't really make sense :(